src/Symfony/Component/AssetMapper/AssetDependency.php

```diff
@@ -19,11 +19,20 @@
 final class AssetDependency
 {
     /**
-     * @param bool $isLazy whether this dependency is immediately needed
+     * @param bool $isLazy Whether the dependent asset will need to be loaded eagerly
+     *                     by the parent asset (e.g. a CSS file that imports another
+     *                     CSS file) or if it will be loaded lazily (e.g. an async
+     *                     JavaScript import).
+     * @param bool $contentDependency Whether the parent asset's content depends
```

Comment:
I don't see any place where you configure a dependency as a content dependency. Any asset compiler retrieving the public path of a dependency needs to do it.

Reply:
ty - great catch